hassio/src/addon-store/hassio-addon-repository.ts

```diff
@@ -21,24 +22,26 @@ class HassioAddonRepositoryEl extends LitElement {
   @property() public hass!: HomeAssistant;
   @property() public repo!: HassioAddonRepository;
   @property() public addons!: HassioAddonInfo[];
+  @property() public filter!: string;
 
   protected render(): TemplateResult | void {
     const repo = this.repo;
-    return html`
-      <div class="card-group">
-        <div class="title">
-          ${repo.name}
-          <div class="description">
-            Maintained by ${repo.maintainer}<br />
-            <a class="repo" href=${repo.url} target="_blank">${repo.url}</a>
+    const addons = this.fuzzySearchAndSort(this.addons, this.filter);
+
+    if (this.filter && addons.length < 1) {
+      return html``;
```

Comment:
Should we print a message that no results are found instead of an empty screen?

Reply:
Now shows 'no results' message for each repository wi no results:

![image](https://user-images.githubusercontent.com/28114703/57560452-6e976580-737e-11e9-9bbd-bf685bdbd9c9.png)